<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaleCommodity</class>
 <widget class="QDialog" name="SaleCommodity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SaleCommodity</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>310</y>
     <width>48</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>销售量：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel">
   <property name="geometry">
    <rect>
     <x>327</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Password">
   <property name="geometry">
    <rect>
     <x>192</x>
     <y>134</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>仓库编号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="commodityNum">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="placeholderText">
    <string>请输入商品编号</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>商品名称：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="commodityName">
   <property name="geometry">
    <rect>
     <x>256</x>
     <y>250</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="placeholderText">
    <string>请输入商品名称</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="commoditySalesUnitPrice">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>370</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入销售单价</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_Account">
   <property name="geometry">
    <rect>
     <x>192</x>
     <y>74</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>仓库所在地：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Title">
   <property name="geometry">
    <rect>
     <x>191</x>
     <y>12</y>
     <width>192</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;销售商品&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="packageLocation">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>74</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>请选择仓库所在地</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>北京</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>天津</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>上海</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>云南</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>四川</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>重庆</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="commoditySales">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>310</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入销售量</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>194</x>
     <y>190</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>商品编号：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="packageNum">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>请选择仓库编号</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>370</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>销售单价:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OK">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
